### PR TITLE
Create wiki.md

### PR DESCRIPTION
**Pull request by alvin21mfmlai**
To kickstart wiki page.

### CODE REVIEW

**Comment by alvin21mfmlai** [comment]
This wiki aims to summarize the class diagrams for the main library packages in TracerX that is built upon KLEE repository. 

**1. Overview**

In TracerX library, the klee and kleaver code is organized as follows:

- lib/Basic: Low level support for both klee and kleaver, independent of LLVM.
- lib/Support: Higher level support that is only used by klee, and using LLVM facilities mainly.
- lib/Expr: The core kleaver expression library.
- lib/Solver: The kleaver solver library.
- lib/Module: klee facilities to function together with LLVM modules, which include the shadow module/instruction structures used during execution.
- lib/Core: The core symbolic virtual machine.

File: Wiki.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (alvin21mfmlai) [inline Wiki.md:1]: This wiki aims to summarize the class diagrams for the main library packages in TracerX that is built upon KLEE repository.

**1. Overview**

In TracerX library, the klee and kleaver code is organized as follows:
- lib/Basic: Low level support for both klee and kleaver, independent of LLVM.
- lib/Support: Higher level support that is only used by klee, and using LLVM facilities mainly.
- lib/Expr: The core kleaver expression library.
- lib/Solver: The kleaver solver library.
- lib/Module: klee facilities to function together with LLVM modules, which include the shadow module/instruction structures used during execution.
- lib/Core: The core symbolic virtual machine

**Review by alvin21mfmlai — COMMENTED** [review]
This wiki aims to summarize the class diagrams for the main library packages in TracerX that is built upon KLEE repository.

1. Overview

In TracerX library, the klee and kleaver code is organized as follows:

lib/Basic: Low level support for both klee and kleaver, independent of LLVM.
lib/Support: Higher level support that is only used by klee, and using LLVM facilities mainly.
lib/Expr: The core kleaver expression library.
lib/Solver: The kleaver solver library.
lib/Module: klee facilities to function together with LLVM modules, which include the shadow module/instruction structures used during execution.
lib/Core: The core symbolic virtual machine

**Comment by rasoolmaghareh** [comment]
Just 2 comments. Please mention that the class diagram is generated from KLEE 1.0. 

Also, will the next step be adding the TX class diagram? or I merge this PR now?

**Comment by alvin21mfmlai** [comment]
Thanks Rasool.

Maybe I have missed out on this information previously, is the current TracerX repository built/generated from the older Klee 1.0 version? In any case, I have added information about the class diagram being generated from Klee 1.0.

Yes, I can add in another TX class diagram to close the loop within a week. You may decide to merge this PR, and I can still edit the wiki.md this week. 

Hope that the above helps.

**Comment by rasoolmaghareh** [comment]
That would be great. Thanks

**Pull request merged**